index.bs

```diff
@@ -84,6 +159,14 @@ interface StorageBucket {
 };
 </xmp>
 
+A {{StorageBucket}} has a <dfn>bucket quota</dfn>, a number-or-null, initially null.
+Specifies the upper limit of usage which can be used by the bucket. It is not guaranteed
+that the specified quota will be available by the user agent.
```

Comment:
How about: The user agent may further limit the realize storage space.

Or maybe add some non-normative text to clarify?